manga_auto_translator/pipeline.py

```diff
@@ -25,6 +29,7 @@ def ocr(self):
         self.ocr_strategy.run(self.scans)
 
     def translation(self):
+        self.translation_strategy.run(self.scans, self.lang_from, self.lang_to)
         pass

```

Comment:
Small detail, but the `pass` keyword can be removed now that there is actual code in the function.

Reply:
Also, we should probably add typing in the function parameters.

Reply:
Fixed